.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ workflows:
       - deploy:
           filters:
             tags:
-              only: /^v\d\.\d\.\d/
+              only: /v\d+(.\d+){2}/
```

Comment:
it should be escaped dot `\.\d+` 
or it can be as it was before just with stars 
/^v\d*\.\d*\.\d*/

Reply:
regex101 to test

Reply:
Gosh! I'll fix it now.